ibis/backends/dask/trace.py

```diff
@@ -0,0 +1,147 @@
+import functools
+import logging
+import traceback
+from datetime import datetime
+
+from ibis.config import get_option, set_option
+from ibis.expr import types as ir
+
+from .dispatcher import TwoLevelDispatcher
+
+"""Module that adds tracing to dask execution.
```

Comment:
same with this, pls create an issue, we want to factor out to a common location (ok for now)

Reply:
https://github.com/ibis-project/ibis/issues/2551